js/freewheel.js

```diff
@@ -146,12 +146,15 @@ OO.Ads.manager(function(_, $) {
       // [PBW-449] To maintain parity with Flash, we allow 'fw_video_asset_id'
       // to be set on the page-level (see 'onPlayerCreated') or fallback to
       // movie metadata, then externalId, and then embed_code.
```

Comment:
Yes, I just simplified the logic. It does the same thing.  The only real change is the one above.